src/theme.js

```diff
@@ -10,7 +10,7 @@ const addAliases = (arr, aliases) =>
     })
   )
 
-export const breakpoints = [32, 40, 48, 64]
+export const breakpoints = [32, 40, 48, 64].map(n => n + 'em')
```

Comment:
Curious about a use case of an array with value of `0`. Would CSS unit `0em` be okay? (Don't think it would break any UI, but just checking on the thought of CSS rule with 0 em unit.
```
[0, 0, 0, 0].map(n => `${n}em`)
```